opencensus/stats/aggregation_data.py

```diff
@@ -183,6 +199,12 @@ def add_sample(self, value):
         self._count_data += 1
         self.increment_bucket_count(value)
 
+        bucket = 0
+        for _ in self.bounds:
```

Comment:
nit: You can do this in `increment_bucket_count()` to avoid looping through the bucket boundaries twice.

Reply:
Done